qiita_db/test/test_sql_connection.py

```diff
@@ -193,173 +195,302 @@ def test_execute_fetchall_with_sql_args(self):
         obs = self.conn_handler.execute_fetchall(sql, (True,))
         self.assertEqual(obs, [['test1', True, 1], ['test2', True, 2]])
 
-    def test_check_queue_exists(self):
-        self.assertFalse(self.conn_handler._check_queue_exists('foo'))
-        self.conn_handler.create_queue('foo')
-        self.assertTrue(self.conn_handler._check_queue_exists('foo'))
 
-    def test_create_queue(self):
-        self.assertEqual(self.conn_handler.queues, {})
-        self.conn_handler.create_queue("toy_queue")
-        self.assertEqual(self.conn_handler.queues, {'toy_queue': []})
+class TestTransaction(TestBase):
+    def test_init(self):
+        obs = Transaction("test_init")
+        self.assertEqual(obs._name, "test_init")
+        self.assertEqual(obs._queries, [])
+        self.assertEqual(obs._results, [])
+        self.assertEqual(obs._index, 0)
+        self.assertTrue(isinstance(obs._conn_handler, SQLConnectionHandler))
+
+    def test_replace_placeholders(self):
```

Comment:
can you add a test here for when there are a few inserts or updates followed by another select, so for example res5a would be next in the results list, as query 5.